package.json

```diff
@@ -57,7 +57,7 @@
     "lodash.camelcase": "^4.3.0",
     "lodash.throttle": "^4.1.1",
     "pcb-stackup": "^4.0.0-next.13",
-    "react": "^15.3.0",
+    "react": "^15.6.2",
```

Comment:
Don't upgrade any packages in this pull-request please. We need to make sure we don't break anything by upgrading so it's best to do upgrades separately.